src/app/components/Tables/ScrutinySettings/EmailScrutinySettingsTable.tsx

```diff
@@ -33,38 +33,46 @@ const ScrutinySettingsTable = ({
-                  <td className='text-center'>{email.adminEmailName}</td>
-                  <td className='text-center'>{email.adminEmailLastName}</td>
-                  <td className='text-center'>{email.adminEmailEmail}</td>
-                  <td className='text-center'>
-                    <span
-                      className={`badge ${
-                        email.adminEmailStatus ? 'badge-light-success' : 'badge-light-warning'
-                      } fs-7 fw-semibold`}
-                    >
-                      {email.adminEmailStatus ? 'ACTIVO' : 'INACTIVO'}
-                    </span>
+              {emailScrutinySettings.length === 0 ? (
+                <tr>
+                  <td colSpan={5} className='text-center'>
+                    No hay resultados disponibles
                   </td>
-                  <td className='text-center'>
-                    <div className='d-flex align-items-center justify-content-center'>
-                      <div
-                        onClick={() => setEmailId(email.adminEmailId, 'update')}
-                        style={{cursor: 'pointer', marginRight: '10px'}}
-                      >
-                        <i className='bi bi-pencil text-primary btn-lg'></i>
-                      </div>
-                      <div
-                        onClick={() => setEmailId(email.adminEmailId, 'delete')}
-                        style={{cursor: 'pointer'}}
+                </tr>
+              ) : (
+                emailScrutinySettings.map((email) => (
+                  <tr className='fw-bold fs-6 text-gray-800' key={email.adminEmailId}>
+                    <td className='text-center'>{email.adminEmailName}</td>
+                    <td className='text-center'>{email.adminEmailLastName}</td>
+                    <td className='text-center'>{email.adminEmailEmail}</td>
+                    <td className='text-center'>
+                      <span
+                        className={`badge ${
+                          email.adminEmailStatus ? 'badge-light-success' : 'badge-light-warning'
+                        } fs-7 fw-semibold`}
                       >
-                        <i className='bi bi-trash text-danger btn-lg'></i>
+                        {email.adminEmailStatus ? 'ACTIVO' : 'INACTIVO'}
+                      </span>
+                    </td>
+                    <td className='text-center'>
+                      <div className='d-flex align-items-center justify-content-center'>
+                        <div
+                          onClick={() => setEmailId(email.adminEmailId, 'update')}
+                          style={{cursor: 'pointer', marginRight: '10px'}}
+                        >
+                          <i className='bi bi-pencil text-primary btn-lg'></i>
+                        </div>
+                        <div
+                          onClick={() => setEmailId(email.adminEmailId, 'delete')}
```

Comment:
please create a function outside of on click, create inline anonymous function is not a good practice, and sometimes those functions could create side effects or not execute